src/main/java/nl/ordina/jobcrawler/service/MatchSkillsService.java

```diff
@@ -69,9 +84,9 @@ private void addStandardSkill(String aSkill) {
     public void insertStandardSkills() {
```

Comment:
This method is not used. Furthermore, it can be better made into an sql file in the test resources.

Reply:
it is used to develop the application, so I will leave it